.github/workflows/release-draft.yml

```diff
@@ -0,0 +1,66 @@
+on:
+  pull_request:
+    types: [opened, synchronize, reopened, edited]
+  # push: to remove above once test is ove
```

Comment:
left here for testing on PRs, but will be changed to only trigger on workflow_dispatch (manually) AND when PR merges to develop